tools/validate_whitespace.py

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env python3
+# /// script
+# requires-python = ">=3.12"
+# ///
```

Comment:
q - this only used if you run it with "uv run", right? So it has no effect on Makefile where it was defined.

Reply:
Yes, serves to document the requirement and make it work with uv.